docs/plugins.rst

```diff
@@ -0,0 +1,231 @@
+.. _plugins:
+
+How to Create Your Own Plugins
+==============================
+
+We built this toolkit for the community, and we knew going in that we couldn't
+possibly build every feature that every user could want, built this thing to be
+pluggable.  You can write your own renderer(s) and use them seamlessly within
```

Comment:
Did you miss something in first line? something feels wrong but you are the native :)